pkg/internal/testing/integration/control_plane.go

```diff
@@ -5,6 +5,7 @@ import (
 	"net/url"
 
 	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/apimachinery/pkg/util/errors"
```

Comment:
mind aliasing the import as `utilerrors`? Thats how its done almost everywhere and the code becomes confusing to read if a given package name refers to different imports in different packages

Reply:
Done, thanks!